### PR TITLE
Use an image w/ node for SourceBuild

### PR DESCRIPTION
**Pull request by wtgodbe**

### CODE REVIEW

**Review by sebastienros — APPROVED** [review]

**Comment by mthalman** [comment]
@MichaelSimons - Are we able to take a dependency on Node?

**Comment by javiercn** [comment]
See https://github.com/dotnet/aspnetcore/issues/46304 and https://github.com/dotnet/aspnetcore/issues/37398 for context.

**Comment by MichaelSimons** [comment]
> @MichaelSimons - Are we able to take a dependency on Node?

Yes, this was the plan discussed at the beginning of .NET 8.0 IIRC.  cc @dotnet/distro-maintainers for awareness and to raise any concerns.

File: .azure/pipelines/ci.yml [hunk 1]
```diff
@@ -734,7 +734,7 @@ stages:
       parameters:
         platform:
           name: 'Managed'
-          container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8'
+          container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.15-WithNode'
```
Comment (javiercn) [inline .azure/pipelines/ci.yml:737]:
```suggestion
          container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8-20240104210103-4893224'
```

**Comment by javiercn** [comment]
@wtgodbe closing this as we integrated the change in the other PR

**Comment by omajid** [comment]
The other PR was https://github.com/dotnet/aspnetcore/pull/53209, right?

**Comment by javiercn** [comment]
@omajid that PR got merged into the branch in https://github.com/dotnet/aspnetcore/pull/53154 which got merged into main yesterday

**Pull request closed without merging**